cdc/sink/mq/dispatcher/topic/expression.go

```diff
@@ -30,6 +30,10 @@ var (
 	schemaRE = regexp.MustCompile(`\{schema\}`)
 	// tableRE is used to match substring '{table}' in topic expression
 	tableRE = regexp.MustCompile(`\{table\}`)
+	// avro has different topic name pattern requirements
```

Comment:
Please provide more detailed comments on it, thank you! For example, the difference from the original?